File: AdvancePython/Day-1 Classes and Objects/Task-3.py
# Task 3: Instantiate object from the class
class Car:
    def __init__(self, make: str, model: str, year: int) -> None:
        self.make: str = make
        self.model: str = model
        self.year: int = year


details = Car("BMW", "M4", 2026)
print(details.make)
print(details.model)
print(details.year)

File: node.js/Day-1/Task-3.js
// Task 3: Instantiate object from the class
class Car {
    constructor(make, model, year) {
        // 'this' keyword refers to the specific object instance being created or used
        this.make = make;
        this.model = model;
        this.year = year;
    }

    details() {
        console.log(`Make: ${this.make}, Model: ${this.model}, Year: ${this.year}`)
    }
}


const c1 = new Car("Audi", "R8", 2025);
c1.details();